Applications/FileIO/TetGenInterface.cpp

```diff
@@ -430,8 +430,8 @@ bool TetGenInterface::parseElements(std::ifstream& ins,
                                     bool region_attribute) const
 {
     std::string line;
-    auto* ids(static_cast<std::size_t*>(
-        alloca(sizeof(std::size_t) * n_nodes_per_tet)));
+    std::vector<std::size_t> ids(n_nodes_per_tet);
```

Comment:
Can `n_nodes_per_tet` be anything other than 4?

Reply:
Could be quadratic...